requirements.yml

```diff
@@ -1,10 +1,14 @@
 ---
 
+- name: openmicroscopy.certbot
+  src: https://github.com/openmicroscopy/ansible-role-certbot/archive/0.1.0.tar.gz
```

Comment:
When do you choose to use galaxy and when not?

Reply:
I use galaxy if it works! Though I haven't retested recently